Sources/TuistGenerator/Mappers/SynthesizedResourceInterfaceProjectMapper.swift

```diff
@@ -112,35 +112,12 @@ public final class SynthesizedResourceInterfaceProjectMapper: ProjectMapping { /
             .appending(component: Constants.DerivedDirectory.name)
             .appending(component: Constants.DerivedDirectory.sources)
 
-        let paths = try paths(for: resourceSynthesizer, target: target, developmentRegion: project.developmentRegion)
-            .filter(isResourceEmpty)
-
-        let templateName: String
-        switch resourceSynthesizer.template {
-        case let .defaultTemplate(name):
-            templateName = name
-        case let .file(path):
-            templateName = path.basenameWithoutExt
-        }
-
-        let renderedInterfaces: [(String, String)]
-        if paths.isEmpty {
-            renderedInterfaces = []
-        } else {
-            let name = target.name.camelized.uppercasingFirst
-            renderedInterfaces = [
-                (
-                    "Tuist\(templateName)+\(name)",
-                    try synthesizedResourceInterfacesGenerator.render(
-                        parser: resourceSynthesizer.parser,
-                        templateString: templateString,
-                        name: target.productName.camelized.uppercasingFirst,
-                        bundleName: project.options.disableBundleAccessors ? nil : "Bundle.module",
-                        paths: paths
-                    )
-                ),
-            ]
-        }
+        let renderedInterfaces = try renderedInterfaces(
+            for: resourceSynthesizer,
+            templateString: templateString,
+            target: target,
+            project: project
+        )
```

Comment:
I had a lint warning about `renderAndMapTarget` function having more than 40 lines.
In order to cut it down, I extracted `renderedInterfaces` in a private function.